HOWTO.md

```diff
@@ -521,10 +542,14 @@ Please mind how the configuration file must be specifically named (e.g. `-` VS
    Files/manufacturing_server.yml](#manufacturing_serveryml) and place it either in
    `/usr/share/fdo`, `/etc/fdo/` or
    `/etc/fdo/manufacturing-serverr.conf.d/`. The paths will be checked in that
-   same order.
-
-3. Execute `fdo-manufacturing-server` or run it as a service, see sample
-   file in
+   same order. 
+   Mention the path for keys and certificates as generated in step1 inside `manufcturing-server.yml`.
+
+3. To execute `fdo-manufacturing-server` as a binary on its own use below command:
+   ```
+   sudo MANUFACTURING_SERVER_CONF=/path/to/manufacturing_server.yml LOG_LEVEL=trace ./target/debug/fdo-manufacturing-server
```

Comment:
This command is very specific and conflicts with the prerequisites of the HOWTO were the binaries have been compiled with the `-release` profile, so this command is not going to work.

Leave it as it was, the environment variables are explained below.